src/index.jsx

```diff
@@ -115,6 +115,27 @@ function AutoHeight({children, element = 'div', className: propClassName = '', .
 
   useEffect(updateHeight)
 
+  // recalculate height on <details> toggle and window resize by forcing rerender
+  const [_, setStateToRerender] = useState()
+  useEffect(() => {
+    let isMounted = true
```

Comment:
is this in case the eventlistners don't get destroyed properly? Or more to handle possible race conditions?

Reply:
yeah, the useless memory leak warning that was removed in React 18 https://github.com/facebook/react/pull/24195 ... but the library supports React 17 too, so let's not scare ourselves when using this library in Stampy 😅

Reply:
and you are correct, it sets state to rerender .. moving code from https://github.com/StampyAI/stampy-ui/blob/3e3ae3ec4e7f4f1ad5a3ef70c3daf424eab92a3e/app/hooks/useRerenderOnResize.ts - but it needs to be new object every time to work, the `set(empty)` was a bug, so let's fix that here in the library and not in distant user code